datalake/src/main/java/com/sequenceiq/datalake/service/validation/cloudstorage/CloudStorageValidator.java

```diff
@@ -37,7 +38,7 @@ public CloudStorageValidator(CredentialToCloudCredentialConverter credentialToCl
 
     public void validate(CloudStorageRequest cloudStorageRequest, DetailedEnvironmentResponse environment) {
         LOGGER.info("Validating cloudStorageRequest: {}", JsonUtil.writeValueAsStringSilent(cloudStorageRequest));
-        if (cloudStorageRequest != null) {
+        if (cloudStorageRequest != null && CloudStorageValidation.ENABLED.equals(environment.getCloudStorageValidation())) {
```

Comment:
please add log it out when the validation is disabled

Reply:
Done in latest push. Thanks!